simphony/io/tests/test_h5_cuds.py

```diff
@@ -346,6 +347,67 @@ def test_mesh_rename(self):
             # "foo" name when adding another mesh
             m = handle.add_mesh(Mesh(name="foo"))
 
+    def test_add_get_delete_lattice(self):
+        lat = Lattice('test_lattice', 'cubic', (1.0, 1.0, 1.0),
+                      (2, 3, 4), (0.0, 0.0, 0.0))
+
+        filename = os.path.join(self.temp_dir, 'test.cuds')
+        with closing(H5CUDS.open(filename, 'w')) as handle:
+            handle.add_lattice(lat)
+            lat2 = handle.get_lattice('test_lattice')
+
+            self.assertEqual(lat2.name, 'test_lattice')
+
+            for N in lat2.iter_nodes():
```

Comment:
See comment in https://github.com/simphony/simphony-common/pull/74/files#r25767404